crypto/pkcs8/pkcs12_test.cc

```diff
@@ -533,7 +551,7 @@ static bssl::UniquePtr<X509> MakeTestCert(EVP_PKEY *key) {
   return x509;
 }
 
-static bool PKCS12CreateVector(std::vector<uint8_t> *out, EVP_PKEY *pkey,
+static bool PKCS12CreateVector(bssl::UniquePtr<PKCS12> *p12, EVP_PKEY *pkey,
```

Comment:
```suggestion
static bool PKCS12CreateVector(bssl::UniquePtr<PKCS12> & p12, EVP_PKEY *pkey,
```

You can just pass this by C++ reference right? Then you don't have to take the address of the UniquePtr every time you pass it in.

Reply:
Agree with both comments, this was mostly me trying to hack something together while allowing the original code to use the same function.